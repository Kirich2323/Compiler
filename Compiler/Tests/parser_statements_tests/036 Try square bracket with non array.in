var
    a : array [1..10] of integer;
    b : integer;
begin
    b := b[1];
end.