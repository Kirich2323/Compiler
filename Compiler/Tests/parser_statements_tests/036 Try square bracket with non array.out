(5 ; 11): Error: Illegal qualifier.